tests/std/tests/P0448R4_iosfwd/test.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <iosfwd>
+
+using namespace std;
+
+// Needs to be separately declared as a simple alias of streampos
+void test_forward_declaration(wstreampos*);
+
+struct test_aliases {
+    static void test_forward_declaration(streampos*);
+
+    static void test_forward_declaration(ios*);
+    static void test_forward_declaration(wios*);
+
+    static void test_forward_declaration(streambuf*);
+    static void test_forward_declaration(istream*);
+    static void test_forward_declaration(ostream*);
+    static void test_forward_declaration(iostream*);
+
+    static void test_forward_declaration(wstreambuf*);
+    static void test_forward_declaration(wistream*);
+    static void test_forward_declaration(wostream*);
+    static void test_forward_declaration(wiostream*);
+
+    static void test_forward_declaration(stringbuf*);
+    static void test_forward_declaration(istringstream*);
+    static void test_forward_declaration(ostringstream*);
+    static void test_forward_declaration(stringstream*);
+
+    static void test_forward_declaration(wstringbuf*);
+    static void test_forward_declaration(wistringstream*);
+    static void test_forward_declaration(wostringstream*);
+    static void test_forward_declaration(wstringstream*);
+
+#if _HAS_CXX23
+    static void test_forward_declaration(spanbuf*);
+    static void test_forward_declaration(ispanstream*);
+    static void test_forward_declaration(ospanstream*);
+    static void test_forward_declaration(spanstream*);
+
+    static void test_forward_declaration(wspanbuf*);
+    static void test_forward_declaration(wispanstream*);
+    static void test_forward_declaration(wospanstream*);
+    static void test_forward_declaration(wspanstream*);
+#endif // _HAS_CXX23
+
+    static void test_forward_declaration(filebuf*);
+    static void test_forward_declaration(ifstream*);
+    static void test_forward_declaration(ofstream*);
+    static void test_forward_declaration(fstream*);
+
+    static void test_forward_declaration(wfilebuf*);
+    static void test_forward_declaration(wifstream*);
+    static void test_forward_declaration(wofstream*);
+    static void test_forward_declaration(wfstream*);
+
+#if _HAS_CXX20
+    static void test_forward_declaration(syncbuf*);
+    static void test_forward_declaration(osyncstream*);
+
+    static void test_forward_declaration(wsyncbuf*);
+    static void test_forward_declaration(wosyncstream*);
+#endif // _HAS_CXX20
+
+#if defined(_CRTBLD)
+    static void test_forward_declaration(ushistream*);
+    static void test_forward_declaration(ushostream*);
+    static void test_forward_declaration(ushfilebuf*);
+#endif // defined(_CRTBLD)

```

Comment:
This can/should be deleted - `_CRTBLD` is defined only when building the STL's separately compiled sources, and is never defined for user/test code. (In fact, these aliases should not be present in the header at all; mentioned in #216 now.)